ProcessLib/NumericsConfig.h

```diff
@@ -129,4 +129,7 @@ static_assert(std::is_same<detail::GlobalMatrixType::IndexType,
 /// GlobalMatrixType::IndexType and the GlobalVectorType::IndexType.
 using GlobalIndexType = detail::GlobalMatrixType::IndexType;
 
+/// A type for scalar number
+using ScalarType = detail::GlobalMatrixType::ScalarType;
```

Comment:
1. For getValues PETScVector, all copies are carried out through PETSc routines.
2. This is not only for VTK,
3. The interfaces  of PETSc vector and matrix enables complex variable, which can be used for solving some problems with complex unknowns. In future, I think, we can come cross such problems.

Under the current development,  PetscScalar is defined as double without an additional preprocess definition. Therefore ScalarType is just removed.